transformations/chinese_simplified_traditional_perturbation/README.md

```diff
@@ -1,28 +1,24 @@
-# Chinese Simplified to Traditional Perturbation (To be updated) 🦎  + ⌨️ → 🐍
-This perturbation adds noise to all types of text sources containing Chinese characters (sentence, paragraph, etc.) proportional to noise erupting 
-from keyboard typos making errors resulting from Chinese characters that have similiar Pinyin. 
+# Chinese Simplified （简体）and Traditional （繁体）Perturbation 🦎  + ⌨️ → 🐍
+This perturbation adds noise to all types of text sources containing Chinese words and characters (sentence, paragraph, etc.) by changing the words and characters between Simplified and Traditional Chinese.
 
 Author name: Timothy Sum Hon Mun
 Author email: timothy22000@gmail.com
 
 ## What type of a transformation is this?
-This transformation perturbes Chinese input text to test robustness. Few Chinese characters that are picked at random will be replaced with characters 
-that have similar pinyin (based on the default Pinyin keyboards in Windows and Mac OS) where the user may accidentally select the wrong character from the returned results. 
-
-It uses a database of 16142 Chinese characters and its associated pinyins to generate the perturbations.
+This transformation perturbes Chinese input text to test robustness. Chinese words or characters that are in Simplified Chinese are picked at random will be replaced with words and characters in
+Traditional Chinese. This transformation can also be performed in the opposite direction. 
 
 ## What tasks does it intend to benefit?
 This perturbation would benefit all tasks which have a sentence/paragraph/document with Chinese characters as input like text classification, 
 text generation, etc.
 
 ## Previous Work
 
-1) Database for Chinese characters: https://github.com/pwxcoo/chinese-xinhua
+1) Open Chinese Convert: https://github.com/BYVoid/OpenCC
 
 ## What are the limitations of this transformation?
-There could be Chinese characters that are not within the database of 16142 characters since there are over 50000 Chinese characters.
-However, the commonly utilized characters in modern Chinese are around 7000 - 8000 characters and most modern Chinese dictionaries will list around 16000 - 20000 characters so the database should cover most cases.
+Although it can only perform transformation to convert between Simplified and Traditional Chinese for now, however it can easily be extended to perform more transformation such as
+Japanese Kanji, character-level and phrase-level conversion, character variant conversion and regional idioms among Mainland China, Taiwan and Hong Kong.
```

Comment:
Thanks for the PR! Is there a particular challenge that makes it hard to include these options in this PR?
(nit pick: Although...~however~ it can easily be)

Reply:
Hi @tongshuangwu , it's quite straightforward to add them since it is just changing an argument :) I'll make the change to reflect it and added comments in the code as well as README to explain which configuration are available